EpikCord/__main__.py

```diff
@@ -21,6 +28,35 @@ async def ready():
 
 client.login()
 """
+__gitignore_template = """# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+# C extensions
+*.so
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+# Our configuration files
+config.py
+"""
+__config_file_template = """
+token = [token]
```

Comment:
i wouldn't store the token in a python file as this isn't a good pratice.

Why? Imagine you are clone the repo of a bot that use such config.py file
Without that file, trying to run the bot will give you an import error, without any explanation and you have to debug it by yourself. Some may argue that there is a not in the readme, but that not enough. It also makes that bot module not testable, or your need to do some trickery to create the missing module in your CI/CD script... no good.
Using a try/except guard around the import,...? Not ideal, and would require t use quit to kill the program on fail.

A `.env`, `config.json` or even a `token.txt` file would be preferred ways, are they are cleaner and less prone to ugly checks.